src/operator/tensor/broadcast_reduce_op_value.cc

```diff
@@ -128,7 +128,7 @@ NNVM_REGISTER_OP(_broadcast_backward)
 .set_attr<FResourceRequest>("FResourceRequest",
   [](const NodeAttrs& attrs) {
     return std::vector<ResourceRequest>{ResourceRequest::kTempSpace};
-  });
+});
```

Comment:
Why removing indent here?

Reply:
It wasn't supposed to be here. It must align with NNVM. For exmaple: https://github.com/apache/incubator-mxnet/blob/master/src/operator/tensor/matrix_op.cc#L446-L491

Reply:
Why must it align with NNVM? There is no open braces for NNVM. This closing braces is matched to the set_attr function. 
Please see https://github.com/apache/incubator-mxnet/blob/master/src/operator/tensor/matrix_op.cc#L751

Reply:
saw other examples in the codebase. Makes sense !